src/Accounts/Authentication.ResourceManager/Authentication.ResourceManager.csproj

```diff
@@ -11,6 +11,10 @@
     <RootNamespace>$(LegacyAssemblyPrefix)Common.Authentication.ResourceManager</RootNamespace>
   </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Microsoft.Azure.Management.SecurityCenter" Version="1.1.1" />
```

Comment:
Why add new package reference here?

Reply:
This was automatically added after updating the SecurityCenter nuget

Reply:
This was automatically added after updating the SecurityCenter nuget. Removed.